aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -132,11 +132,43 @@ extern "C" void dgetrf_(int *m, int *n, double *a, int *lda, int *ipiv, int *inf
 extern "C" void sgetrf_(int *m, int *n, float *a, int *lda, int *ipiv, int *info);
 
 // potrs
+#if defined(_WIN32) && defined(_M_ARM64)
+#if AT_BUILD_WITH_APL()
```

Comment:
```suggestion
// TODO: Add reference to ARM Performance Library issue explaining why those are needed
```

Reply:
zpotrs, cpotrs, dpotrs, spotrs, zunqmr, cunmqr, dormqr, and sormqr functions are not properly usable on Windows Arm64. Other platforms should be okay, hence the if guard for Windows Arm64.
We opened an issue on ARM Platform ( https://community.arm.com/support-forums/f/high-performance-computing-forum/56512/unable-to-use-lapack---potrs-functions) and mentioned it as a comment.